src/main/scala/Syntax.scala

```diff
@@ -173,7 +173,7 @@ object Syntax {
   case object CEmpty extends Command
 
   sealed trait Definition extends Positional
-  case class FuncDef(id: Id, args: List[Decl], body: Command) extends Definition
+  case class FuncDef(id: Id, args: List[Decl], bodyOpt: Option[Command]) extends Definition
```

Comment:
A comment here about why the body is optional seems useful.

Reply:
Fixed.